make/RunTestsPrebuilt.gmk

```diff
@@ -159,17 +159,26 @@ ifeq ($(UNAME_OS), CYGWIN)
   OPENJDK_TARGET_OS_TYPE := windows
   OPENJDK_TARGET_OS_ENV := windows.cygwin
 else
-  OPENJDK_TARGET_OS_TYPE:=unix
-  ifeq ($(UNAME_OS), Linux)
-    OPENJDK_TARGET_OS := linux
-  else ifeq ($(UNAME_OS), Darwin)
-    OPENJDK_TARGET_OS := macosx
+  ifeq ($(UNAME_OS), MINGW64)
```

Comment:
Instead of nesting, this inside the else body, creating more nesting below, make supports 
```
else ifeq ...
```
By using that you can keep the else body with `unix` on the same indentation level.